bin/run-checks.py

```diff
@@ -1,50 +1,74 @@
 import os
 import sys
+import traceback
 
 from pprint import pformat
 
 from data_checks.check_datapackage import run_checks
 
 
 def run_report(filepath):
-    print("===== Checking {} =====".format(filepath))
-    report = run_checks(filepath)
+    print("\n===== Checking {} =====".format(filepath))
+    try:
+        report = run_checks(filepath)
+    except Exception as e:
+        print("===== Goodtables checks resulted in an error =====")
+        traceback.print_exc(file=sys.stdout)
+        if isinstance(e, KeyError):
+            print(
+                "\n===== It looks like the dataset might contain unexpected columns. ====="
```

Comment:
Is it easy to make this more restrictive? Can you easily check that the error message is "field" so it's more specific to this particular KeyError?

Reply:
Looks like it is :+1: